src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="bigSearch">
+      <div class="icon1">
+        <img src="images/Search.svg" alt="searchIcon">
+      </div>
+      <form
+        data-qa="big"
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post">
```

Comment:
```suggestion
      <form
        data-qa="big"
        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
        method="post"
      >
```
check code style in otther

Reply:
![Screenshot from 2023-04-04 15-27-22](https://user-images.githubusercontent.com/33029864/229791246-62ae49d1-87be-4925-896f-3dd8f3b87e2b.png)